modules/apps/frontend-theme/test.properties

```diff
@@ -24,4 +25,5 @@
 ## Testray
 ##
 
-    testray.main.component.name=Theme
\ No newline at end of file
+    testray.main.component.name=Theme
+    testray.main.component.name=Navigation
```

Comment:
@Esther-OC, This line needs to be removed. frontend-theme module is owned by FI team so should have main.component.name set to a FI owned component ie: Theme. Not Navigation which is owned by echo team. Please send another pull to fix this.